- fixes #4694: documents batching in combinatino with throttling (#9560)
microsoftgraph/microsoft-graph-docs: concepts/throttling.md

>[!NOTE]
>[Best practices for discovering files and detecting changes at scale](https://docs.microsoft.com/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online) describes best practices in details.

## Throttling and batching

> [JSON batching](./json-batching.md) allows you to optimize your application by combining multiple requests into a single JSON object. Requests in a batch are evaluated individually against throttling limits and should any request be exceed limits, it will fail individually with a `status` of `429` and an error similar to the one provided above. The batch itself will fail with status code `424` (Failed Dependency). It is possible multiple requests get throttled in a single batch. You should retry each failed request from the batch after the value provided in the `retry-after` response header from the JSON content. You may retry all the failed requests in a new batch after the longest `retry-after` value.

Reviewer: ```suggestion
[JSON batching](./json-batching.md) allows you to optimize your application by combining multiple requests into a single JSON object. Requests in a batch are evaluated individually against throttling limits and if any request exceeds the limits, it fails with a `status` of `429` and an error similar to the one provided above. The batch itself fails with a status code of `424` (Failed Dependency). It is possible for multiple requests to be throttled in a single batch. You should retry each failed request from the batch using the value provided in the `retry-after` response header from the JSON content. You may retry all the failed requests in a new batch after the longest `retry-after` value.
```
